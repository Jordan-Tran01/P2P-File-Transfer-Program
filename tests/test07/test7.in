CONNECT 127.0.0.1:9856
ADDPACKAGE pkg1.bpkg
FETCH 127.0.0.1:9856 3cf007c14ded16ab85d168fcf9d9b20effef7a0b8f89524d72eeaea97832a3193f9aa9528ebd0 c22250db8c3cdb65bc5e722dc408651cbb09a87ec3fb1f0555bb26a1fab96892