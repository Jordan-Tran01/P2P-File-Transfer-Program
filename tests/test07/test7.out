Unable to request chunk, package is not managed
Connection established with peer
Failed to create file 'btide_test1/btide_test1/pkg1.data'
Unable to request chunk, package is not managed
